client/src/components/Form/Elements/FormSelect.vue

```diff
@@ -0,0 +1,182 @@
+<script setup lang="ts">
+import { computed, onMounted, watch, type ComputedRef } from "vue";
+import Multiselect from "vue-multiselect";
+
+type SelectValue = string | null;
+
+interface SelectOption {
+    label: string;
+    value: SelectValue;
+}
+
+const props = withDefaults(
+    defineProps<{
+        multiple?: boolean;
+        optional?: boolean;
+        options: Array<[string, string]>;
+        value?: Array<string> | string;
+    }>(),
+    {
+        multiple: false,
+        optional: false,
+        value: null,
```

Comment:
isn't that a type error ? null isn't `Array<string> | string | undefined`

Reply:
Explicitly adding `null` as type to the list throws an error, maybe due to the `?` which already declares that prop as optional.